artiq/language/environment.py

```diff
@@ -392,13 +394,14 @@ def analyze(self):
 
 
 class EnvExperiment(Experiment, HasEnvironment):
-    """Base class for top-level experiments that use the ``HasEnvironment``
-    environment manager.
+    """Base class for top-level experiments that use the
+    :class:`~artiq.language.environment.HasEnvironment` environment manager.
 
     Most experiments should derive from this class."""
     def prepare(self):
-        """The default prepare method calls prepare for all children, in the
-        order of instantiation, if the child has a prepare method."""
+        """This default prepare method calls :meth:`~artiq.language.environment.Experiment.prepare`
+        for all children, in the order of instantiation, if the child has a
+        :meth:`~artiq.language.environment.Experiment.prepare` method."""
```

Comment:
And put the final """ onto a line by itself if the docstring is longer than one line.